packages/devtools_app/lib/src/info/flutter/info_screen.dart

```diff
@@ -1,204 +0,0 @@
-}
-
-class InfoScreenBody extends StatefulWidget {
-  const InfoScreenBody();
-
-  @override
-  _InfoScreenBodyState createState() => _InfoScreenBodyState();
-}
-
-class _InfoScreenBodyState extends State<InfoScreenBody> {
-  FlutterVersion _flutterVersion;
-
-  InfoController _controller;
-
-  @override
-  void initState() {
-    super.initState();
-    _controller = InfoController(
-      onFlutterVersionChanged: (flutterVersion) {
-        if (!mounted) return;
-        setState(
-          () {
-            _flutterVersion = flutterVersion;
-          },
-        );
-      },
-    )..entering();
-  }
-
-  @override
-  void dispose() {
-    _controller?.dispose();
-    super.dispose();
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    final textTheme = Theme.of(context).textTheme;
-    return Column(
-      crossAxisAlignment: CrossAxisAlignment.start,
-      children: [
-        Text(
-          'Version Information',
-          style: textTheme.headline5,
-        ),
-        const PaddedDivider(padding: EdgeInsets.only(top: 4.0, bottom: 0.0)),
-        if (_flutterVersion != null)
-          _VersionInformation(_flutterVersion),
-        const Padding(padding: EdgeInsets.only(top: defaultSpacing)),
-        // TODO(devoncarew): Move this information into an advanced page.
-        Text(
-          'Dart VM Flag List',
-          style: textTheme.headline5,
-        ),
-        const PaddedDivider(padding: EdgeInsets.only(top: 4.0, bottom: 0.0)),
-        Expanded(
-          child: ValueListenableBuilder<FlagList>(
-            valueListenable: _controller.flagListNotifier,
-            builder: (context, flagList, _) {
-              if (flagList == null || flagList.flags.isEmpty) {
-                return const SizedBox();
-              }
-              return _FlagList(flagList);
-            },
-          ),
-        ),
-      ],
-    );
-  }
-}
-
-class _VersionInformation extends StatelessWidget {
-  const _VersionInformation(this.flutterVersion);
-
-  final FlutterVersion flutterVersion;
-
-  @override
-  Widget build(BuildContext context) {
-    const boldText = TextStyle(fontWeight: FontWeight.bold);
-
-    final versions = {
-      'DevTools': devtools.version,
-      'Flutter': flutterVersion.version,
-      'Framework': flutterVersion.frameworkRevision,
-      'Engine': flutterVersion.engineRevision,
-      'Dart': flutterVersion.dartSdkVersion,
-    };
-
-    return Column(
-      key: InfoScreen.flutterVersionKey,
-      children: [
-        for (var name in versions.keys)
-          Padding(
-            padding: const EdgeInsets.only(left: 8.0, bottom: 8.0),
-            child: Row(
-              children: [
-                Text(name, style: boldText),
-                const SizedBox(width: 8.0),
-                Text(versions[name]),
-              ],
-            ),
-          ),
-      ],
-    );
-  }
-}
-
-class _FlagList extends StatelessWidget {
```

Comment:
Showing the flag list has been useful for us a few times in helping users debug issues. 
My preference would be to not delete this UI. It is not nearly as advanced vm author specific as other functionality that would be exposed only in advanced mode.
For example, I would like all offline snapshots to start including a snapshot of the flag data so that when we debug a user issue we can see exactly which flags were set.
Example of a similar page in chrome:
chrome://flags/

Reply:
I agree. I'd be in favor of creating a "Flags" page with a flag icon to house this UI. This page will probably need to be disabled for dart web apps, as the info page was.

Reply:
Ah, OK. I may restore the info page then, or a new similar Flags page, with just the DevTools version removed.